payroll/serializers.py

```diff
@@ -76,14 +76,14 @@ def employer_queryset(self):
 
     @property
     def employer_salaries(self):
-        if not hasattr(self, '_salaries'):
+        if not hasattr(self, '_employer_salaries'):
             self._employer_salaries = self.instance.get_salaries(year=self.context['data_year'])
         return self._employer_salaries
 
     @property
     def employer_salary_count(self):
         if not hasattr(self, '_employer_salary_count'):
-            self._employer_salary_count = self._employer_salaries.count()
+            self._employer_salary_count = self.employer_salaries.count()
```

Comment:
This was a typo! When I took out the top_salaries ORM query, i.e., the `self.employer_salaries` call, which set this underscored property, this one started to fail, because it became the first (incorrect) reference. Also corrected the definition above.